src/components/item/item-reorder.scss

```diff
@@ -16,12 +16,18 @@ ion-reorder {
   font-size: 1.7em;
   opacity: .25;
 
-  transform: translate3d(160%, 0, 0);
```

Comment:
I think it is better to revert and have right side by default, then add: "reorder-end" dynamically